corehq/apps/data_interfaces/deduplication.py

```diff
@@ -106,3 +107,11 @@ def backfill_deduplicate_rule(domain, rule):
             locked_for_editing=False,
             last_run=now,
         )
+
+
+def get_dedupe_xmlns(rule):
+    return f"{DEDUPE_XMLNS}__{rule.name}-{rule.case_type}"
```

Comment:
I'm not sure what a name will look like but it would probably be safest to slugify it first